be/src/service/service_be/lake_service.cpp

```diff
@@ -450,6 +450,28 @@ void LakeServiceImpl::delete_txn_log(::google::protobuf::RpcController* controll
     latch.wait();
 }
 
+namespace drop_table_helper {
+static bthread::Mutex g_mutex;
+static std::unordered_set<std::string> g_paths;
+
+// Return true if the container doesn't already contain "path".
```

Comment:
according to the method name, `returns true` usually mean the dedup hits, which is the inverted case of current implementation. :)

Reply:
Updated